pkg/wasm/httpfetcher.go

```diff
@@ -51,14 +51,16 @@ func (f *HTTPFetcher) Fetch(url string, timeout time.Duration) ([]byte, error) {
 		}
 	}
 	attempts := 0
+	retryBackoff := f.retryBackoff
 	var lastError error
 	for attempts < 5 {
 		attempts++
 		resp, err := c.Get(url)
 		if err != nil {
 			lastError = err
 			wasmLog.Debugf("wasm module download request failed: %v", err)
-			time.Sleep(f.retryBackoff)
+			time.Sleep(retryBackoff)
+			retryBackoff <<= 1
```

Comment:
I think we have min duration / max duration in the code base somewhere, could use that.

50 ms * 32 (5 hardcoded) retries is not too bad though.

Reply:
Yeah let's use `github.com/cenkalti/backoff` package here, which is also used by other part of the code base.